engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/identity/CheckPasswordPolicyResultDto.ftl

```diff
@@ -0,0 +1,17 @@
+<@lib.dto>
```

Comment:
Document in the ticket and comment here that the inheritance in Rest API is different

Reply:
Actually, that's an oversight on my part. It should inherit.